packages/client-search/src/methods/index/replaceAllSynonyms.ts

```diff
@@ -9,11 +9,14 @@ export const replaceAllSynonyms = (base: SearchIndex) => {
   return (
     synonyms: readonly Synonym[],
     requestOptions?: RequestOptions &
-      Pick<SaveSynonymsOptions, Exclude<keyof SaveSynonymsOptions, 'replaceExistingSynonyms'>>
+      Pick<
+        SaveSynonymsOptions,
+        Exclude<keyof SaveSynonymsOptions, 'clearExistingSynonyms' | 'replaceExistingSynonyms'>
+      >
   ): Readonly<WaitablePromise<SaveSynonymsResponse>> => {
     return saveSynonyms(base)(synonyms, {
       ...requestOptions,
-      replaceExistingSynonyms: true,
```

Comment:
if someone called replaceAllSynonyms with replaceExistingSynonyms: false, this would be a breaking change, however it's an excluded parameter, therefore not using typescript and not supported usage (wrong parameters)